Fixed #15799 -- Doc'd that Storage._open() should raise FileNotFoundError when file doesn't exist. (#17144)
django/django: docs/howto/custom-file-storage.txt

Called by ``Storage.open()``, this is the actual mechanism the storage class
uses to open the file. This must return a ``File`` object, though in most cases,
you'll want to return some subclass here that implements logic specific to the

> backend storage system. This will raise a ``FileNotFoundError`` exception if the
> file does not exist. To catch: ::
> 
>     try:
>         f = myStorage.open()
>     except FileNotFoundError as e:
>         print(f"File {path} not found!", file=sys.stderr)

Reviewer: This ticket is about an exception that should be raised by `_open()`, not how to catch exceptions from `open()`. Maybe:
```suggestion
backend storage system. The :exc:`FileNotFoundError` exception should be raised
when a file doesn't exist.
```

Author: Thank you for your feedback.  I misunderstood the original ask.  I have reflected your feedback in the most recent commit.

    from django.core.files.storage import storages

>     example_storage = storages["example"]

Reviewer: Revert unrelated change.

Author: I originally added to the end of the file, deleted what I had, but left a trailing empty line by accident.  I then removed the extra line in a later commit.  The code has not been modified in any way.
